draft-ietf-mptcp-rfc6824bis-11.xml

```diff
@@ -1430,26 +1430,24 @@ break the multipath connections, it will just reduce the opportunity to open mul
             this limit is implementation specific. A RECOMMENDED number is 3. If no TCP RST
             is received in response, Host A SHOULD send a TCP RST with the MP_FASTCLOSE option
 	    itself when it releases state in order to clear any remaining state at middleboxes.</t>
-
-	    <t>Upon receipt of a RST with MP_FASTCLOSE, containing the valid key,
-	    Host B tears down all subflows. Host B can now close the whole MPTCP
-	    connection (it transitions directly to CLOSED state).</t>
           </list>
         </t>
+
+	<t>If however host A decides to force the closure by using Option R and
+	sending a RST with the MP_FASTCLOSE option, Host B will act as follows:
+	Upon receipt of a RST with MP_FASTCLOSE, containing the valid key,
+	Host B tears down all subflows. Host B can now close the whole MPTCP
```

Comment:
Should we require a RST from Host B on all subflows too?

Reply:
Might be useful! I will add this to the text.